FAIL
Kernel panic in run: PANIC at ../../threads/thread.c:291 in thread_current(): assertion `is_thread (t)' failed.
Call stack: 0x80042141ba 0x8004207094 0x800420ab8f 0x800420a649 0x8004207c06 0x8004206b50 0x8004206068
Translation of call stack:
0x00000080042141ba: debug_panic (lib/kernel/debug.c:32)
0x0000008004207094: thread_current (threads/thread.c:292)
0x000000800420ab8f: lock_held_by_current_thread (threads/synch.c:314)
0x000000800420a649: lock_acquire (threads/synch.c:196)
0x0000008004207c06: allocate_tid (threads/thread.c:643)
0x0000008004206b50: thread_init (threads/thread.c:135)
0x0000008004206068: main (threads/init.c:83)
